src/cozy/wrappers/angular-wrapper.component.ts

```diff
@@ -1,203 +1,209 @@
-    }
-
-    /******************/
-    /* Props Bindings */
-    /******************/
-
-    /**
-     * Get props that will be injected into the ReactWrapper
-     *
-     * @param checkExtensionInstalled Set to true if the component needs to query the installation state online (ex: for installation page).
-     * Else installation state will be set to true without any query
-     *
-     * @returns props to be injected into the ReactWrapper
-     */
-    async getReactWrapperProps(checkExtensionInstalled = false): Promise<ReactWrapperProps> {
-        const client = this.clientService.GetClient();
-
-        const hasHint = checkExtensionInstalled
-            ? await this.fetchHintExists(client)
-            : true;
-
-        const bitwardenData = {
-            extension_installed: hasHint,
-        };
-
-        return {
-            client: client,
-            bitwardenData: bitwardenData,
-            vaultData: this.getVaultData(),
-        };
-    }
-
-    protected getVaultData(): VaultData {
-        const importService = new ImportService(
-            this.cipherService,
-            this.folderService,
-            this.apiService,
-            this.i18nService,
-            this.collectionService,
-            this.platformUtilsService,
-            this.cryptoService
-        );
-
-        const containerService = new ContainerService(this.cryptoService);
-
-        const vaultData = {
-            apiService: this.apiService,
-            environmentService: this.environmentService,
-            authService: this.authService,
-            syncService: this.syncService,
-            cryptoService: this.cryptoService,
-            cipherService: this.cipherService,
-            userService: this.userService,
-            collectionService: this.collectionService,
-            passwordGenerationService: this.passwordGenerationService,
-            containerService: containerService,
-            vaultTimeoutService: this.vaultTimeoutService,
-            importService: importService,
-            utils: Utils,
-        };
-
-        return vaultData;
-    }
-
-    protected async fetchHintExists(client: CozyClient) {
-        try {
-            await client
-                .getStackClient()
-                .collection('io.cozy.settings')
-                .get('hint');
-
-            return true;
-        } catch (e) {
-            return false;
-        }
-    }
-
-    /**********/
-    /* Render */
-    /**********/
-
-    protected getRootDomNode() {
-        const node = document.getElementById(this.rootDomID);
-        invariant(node, `Node '${this.rootDomID} not found!`);
-        return node;
-    }
-
-    protected renderReact() {
-        throw new Error('should be overridden');
-    }
-
-    protected renderReactIfMounted() {
-        if (this.isMounted()) {
-            this.renderReact();
-        }
-    }
-
-    private isMounted(): boolean {
-        return !!this.rootDomID;
-    }
-}
+import {
```

Comment:
Care, you did not configure your IDE env to auto-detect line endings. Here you changed all line endings. That's why all the file's lines are different